### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by bkabrda**
- unify log messages to start lowercase
- provide more info on what is going on

Hopefully this will make logs more useful (and also it should make them look a bit better :))

### CODE REVIEW

File: atomic_reactor/build.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def _ensure_is_built(self):
         :return: None
         """
         if not self.is_built:
-            logger.error("Image is not built yet!")
+            logger.error("Image '%s' is not built yet!", self.image)
```
Comment (twaugh) [inline atomic_reactor/build.py]: Looks like there are still a few places that have capital letters at the start.

File: atomic_reactor/build.py [hunk 2]
```diff
@@ -48,7 +49,7 @@ def _ensure_not_built(self):
         :return: None
         """
         if self.is_built:
-            logger.error("Image is already built!")
+            logger.error("Image '%s' is already built!", self.image)
```
Comment (twaugh) [inline atomic_reactor/build.py]: Leading capital

File: atomic_reactor/buildimage.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def create_image(self, df_dir_path, image, use_cache=False):
         :param image:
         :return:
         """
-        logger.debug("df_dir_path = '%s', image = '%s'", df_dir_path, image)
+        logger.debug("Creating build image: df_dir_path = '%s', image = '%s'", df_dir_path, image)
```
Comment (twaugh) [inline atomic_reactor/buildimage.py]: Leading capital

**Comment by bkabrda** [comment]
Thanks for the catches, Tim. I fixed these in the second commit.

**Comment by bkabrda** [comment]
In some cases, yes. The thing is, the new info messages don't contain all the information that's also debug-logged for all the functions (it wouldn't make sense to do so). So I'd rather log all arguments for all core functions separately, even if they're logged twice on debug.

**Comment by jpopelka** [comment]
OK, feel free to rebase and merge.

**Comment by TomasTomecek** [comment]
lower case messages; Slavek, you are my hero! Anyway, this will be really handy.

**Pull request merged**